standards-catalogue/.schema/standards.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://github.com/alphagov/data-standards-authority/blob/main/standards-catalogue/.schema/standards.json",
+  "title": "Data Standards Authority Standards objects",
+  "description": "A JSON Schema to manage the format of Standards objects",
+  "type": "object",
+  "required": [
+    "name",
+    "contraction",
+    "specification_url",
+    "maintainer_id",
+    "endorsement_status_events",
+    "tags",
+    "license_id"
+  ],
+  "properties": {
+    "identifier": {
+      "$comment": "The unique identifier assigned to this standard",
+      "type": "string"
+    },
+    "name": {
+      "$comment": "The title or name of the standard",
+      "type": "string",
+      "maxLength": 70
+    },
+    "contraction": {
+      "$comment": "A shorter way of referring to the standard, for instance via an acronym",
+      "type": "string",
+      "maxLength": 10
+    },
+    "specification_url": {
+      "$comment": "A link to a publicly accessible specification for the standard",
+      "type": "string",
+      "pattern": "^https?://.*"
+    },
+    "maintainer_id": {
```

Comment:
just checking, is maintainer the new name for organisation?

Reply:
Yep